Repositories/MakRevisionRepository.php

```diff
@@ -32,7 +32,8 @@ public function addRevision(string $type, string $objectId, string $entityId = n
         $statement = $this->_em
             ->getConnection()
             ->prepare(
-                'REPLACE INTO mak_revision (`type`, `id`, `entity_id`, `changed`) VALUES (:type, :id, :entityId, NOW())'
+                'REPLACE INTO s_plugin_makaira_connect_revision (`type`, `id`, `entity_id`, `changed`)
+                VALUES (:type, :id, :entityId, NOW())'
             );
         $statement->execute(['type' => $type, 'id' => $objectId, 'entityId' => $entityId]);
```

Comment:
exec function is showing as deprecated on my IDE

Reply:
See my comment above.